test/instrumentation/modules/tedious.js

```diff
@@ -33,10 +33,7 @@ if (semver.gte(version, '4.0.0')) {
       },
       options: {
         database: 'master',
-        encrypt: false,
-        cryptoCredentialsDetails: {
-          ciphers: 'RC4-MD5'
-        }
+        encrypt: false
```

Comment:
Is this not needed anymore? 🤔

Reply:
`RC4-MD5` fails on Node.js 12 because it's not supported anymore. I couldn't figure out what else to use, so I tried to remove it and it worked just fine. That's one of the reasons why I wanted to run the TAV tests so we could see if this would also work in older versions of tedious.

Reply:
Looks like all TAV tests besides tedious pass.. but the tedious failures seem odd. I'll just try to re-run them in case they were one-offs.

Reply:
🤦‍♂ obviously the tedious TAV tests are unrelated to this change as the change only affects AppVeyor. Don't know what I was thinking there... so the TAV tests must be unrelated